air-quality-ui/system_tests/utils/helper_methods.ts

```diff
@@ -2,10 +2,10 @@ import { type Locator, type Page } from '@playwright/test'
 
```

Comment:
Is this function to be used across several pages? If so, add it to the `base_page.ts` and then we don't need helper methods